app/src/main/java/com/nsc/covidscore/MainActivity.java

```diff
@@ -54,4 +65,46 @@ public void onChanged(@Nullable final Location locationFromDb) {
 
         Log.d(TAG,"onCreate invoked");
     }
-}
\ No newline at end of file
+
+    @Override
+    protected void onStop () {
+        super.onStop();
+        if (queue != null) {
+            requestManager.getRequestQueue().cancelAll(TAG);
+        }
+    }
+
+    private void makeApiCalls() {
+        Requests.getCounty(this, "king,washington", new VolleyJsonCallback() {
+            @Override
+            public void getJsonData(JSONObject response) {
+                Log.d(TAG, "getJsonData: county " + response);
```

Comment:
Nice! Just having the response print to logs so I can see what format it comes in. That'll make it much easier for me to hook this up to Room! : )